app/lib/proxy_config_affecting_changes.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ProxyConfigAffectingChanges
+  module ProxyRuleExtension
+    extend ActiveSupport::Concern
+
+    included do
+      include ProxyConfigAffectingChanges
+
+      after_commit :issue_proxy_affecting_change_events
+
+      def issue_proxy_affecting_change_events
+        case owner
+        when Proxy
+          issue_proxy_affecting_change_event(owner)
+        when BackendApi
+          owner.proxies.each(&method(:issue_proxy_affecting_change_event))
+        end
+      end
+    end
+  end
+
+  module ProxyExtension
+    extend ActiveSupport::Concern
+
+    PROXY_CONFIG_AFFECTING_ATTRIBUTES = %w[policies_config].freeze # TODO: add more attributes here
+
+    included do
+      include ProxyConfigAffectingChanges
+
+      after_commit :issue_proxy_affecting_change_events, on: :update
+
+      def issue_proxy_affecting_change_events
+        changes_attributes = previous_changes.keys
+        return if changes_attributes.include?('created_at')
```

Comment:
```suggestion
        return if created_at_changed? || (changes_attributes & PROXY_CONFIG_AFFECTING_ATTRIBUTES).empty?
```

Reply:
No, it's not `changed_attributes`, it's `changes_attributes`, a variable defined in the line before.